recipes/centos_dotnet20/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2012-2017 Red Hat, Inc
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+FROM registry.centos.org/che-stacks/centos-jdk8:latest
```

Comment:
@dharmit shouldn't we use image `centos-stack-base`  as base stack? The base stack has everything is needed to run on Che (jdk included). `centos-jdk8` adds maven and tomcat on top of the base stack. Do we need them for .NET?

Reply:
Makes sense. I'll change it.